### PR TITLE
Add listClassesInDirectories function

### PR DESCRIPTION
**Pull request by azjezz**
allowing to list classes in multiple directories at once

### CODE REVIEW

File: src/functions.php [hunk 1]
```diff
@@ -26,3 +32,17 @@ function listClassesInDirectory(string $directory): iterable
         yield $class->getName();
     }
 }
+
+/**
+ * get a list of all classes in the given direcotory.
+ *
+ * Based on: https://github.com/Roave/BetterReflection/blob/396a07c9d276cb9ffba581b24b2dadbb542d542e/demo/parsing-whole-directory/example2.php.
```
Comment (WyriHaximus) [inline src/functions.php]: Could you remove this line? Only makes sense for the function above.

**Review by WyriHaximus — CHANGES_REQUESTED** [review]
Thank you for the PR, this looks great, two request tho. Could you add an example to the readme, and remove the `Based on` link on the bottom function?

P.S. Running `make` runs all the checks also ran on `Travis` and `AppVeyor`

File: src/functions.php (hunk 1 above)
Comment (azjezz) [reply src/functions.php]: sure

File: README.md [hunk 2]
```diff
@@ -18,8 +18,20 @@ composer require wyrihaximus/list-classes-in-directory
...
 
 ```php
+
+use function WyriHaximus\listClassesInDirectories;
+use function WyriHaximus\listClassesInDirectory;
+
 // $classes now contains a list of full qualified class names from __DIR__
 $classes = listClassesInDirectory(__DIR__);
+
+// you can also get q list of classes from multiple directories
```
Comment (WyriHaximus) [inline README.md]: Assuming this should be `get a list`?

File: src/functions.php (hunk 1 above)
Comment (WyriHaximus) [reply src/functions.php]: Cheers :+1:

File: README.md (hunk 2 above)
Comment (azjezz) [reply README.md]: i just switched from azerty to a qwerty keyboard .. ugh

Comment (WyriHaximus) [reply README.md]: Oef getting used to that takes time, good luck!

File: src/functions.php [hunk 3]
```diff
@@ -9,13 +9,19 @@
...
+ * get a list of all classes in the given direcotories.
+ *
  * Based on: https://github.com/Roave/BetterReflection/blob/396a07c9d276cb9ffba581b24b2dadbb542d542e/demo/parsing-whole-directory/example2.php.
+ * 
+ * @param array<int, string> $directories
+ *
+ * @return iterable
  */
-function listClassesInDirectory(string $directory): iterable
+function listClassesInDirecories(string ...$directories): iterable
```
Comment (WyriHaximus) [inline src/functions.php]: There is a `t` missing in `Directories`, on line 45 as well

Comment (azjezz) [reply src/functions.php]: everything is fixed, i though adding 1 function would be easy from the webflow - i was wrong

Comment (WyriHaximus) [reply src/functions.php]: Haha been there done that, doesn't always work out as we intend to :zipper_mouth_face:

**Comment by azjezz** [comment]
@WyriHaximus  ready to be merged !

**Review by WyriHaximus — APPROVED** [review]

**Comment by WyriHaximus** [comment]
Awesome! Thank you :tada:

**Pull request merged**